sonarqube/assets/configuration/spec.yaml

```diff
@@ -116,26 +115,43 @@ files:
         options:
           - name: limit
             description: |
-              Maximum number of components to be 'autodiscovered'
+              Maximum number of components to be 'autodiscovered'.
             value:
               type: integer
               example: 10
           - name: include
             description: |
-              Mapping of regular expressions keys and component config values that will be 'autodiscovered'
+              Mapping of regular expressions keys and component config values that will be 'autodiscovered'.
+              
+              The top-level keys are the regular expressions patterns for the desired component names. 
+
+              include:
+                my_project*:
+                  <OPTION_1>: ...
+                issue*:
+                  <OPTION_1>: ...
+                  <OPTION_2>: ...
+
+              The available include component object options are:
+
+                tag:      Custom tag for the matching components.
+                include:  List of metrics regular expressions patterns to include from the matching components.
+                exclude:  List of metrics regular expressions patterns to exclude from the matching components.
             value:
               type: object
-            properties:
-            - name: tag
-              type: string
-            - name: include
-              type: array
-              items:
-                type: string
-            - name: exclude
-              type: array
-              items:
-                type: string
+              additionalProperties:
```

Comment:
I wonder if this definition should be the same as the corresponding one in components

Reply:
Does `components_discovery` support discovering all projects when it has a `None` value? 

I.e. 

```
components_discovery:
  include:
    <project>:
```

I tested it in ddev and it didn't collect any metrics.

Reply:
Can you show me the config you tested?